docs/DeveloperGuide.md

```diff
@@ -346,40 +346,590 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
+  3b2. User requests to move to the Personal list or just delete<br>
+  3b3. Dueue moves all unfinished tasks to Personal list or delete all unfinished tasks and deletes the list<br>
+  Use case ends.
+
+#### Use case: View different colors for different priority levels
+
+**MSS**
+
+1. User requests to display different colors for different priority levels
+2. Dueue displays priority levels with different colors <br>
+Use case ends.
+
+#### Use case: View the same colors for different priority levels
+
+**MSS**
+
+1. User requests not to display different colors for diffrent priority levels.
+2. Dueue displays priority levels with same color <br>
+Use case ends.
+
+#### Use case: Set priority level for a task
+
+**MSS**
+
+1. User requests to change priority of a certain task
+2. Dueue asks for new priority level
+3. User types in the new level
+4. Dueue display this tasks with new level <br>
+Use case ends.
+
+**Extensions**
+
+1a. The given index is invalid
+
+> 1a1. Dueue shows an error message <br>
+  Use case resumes at step 1
+
+3a. The priority level is invalid
+
+> 3a1. Dueue shows an error message <br>
+  Use case resums at step 2
+
+#### Use case: "Star" a task to be favorite
+
+**MSS**
+
+1. User requests to "star" a certain task
+2. Dueue displays the task with star <br>
+Use case ends.
+
+**Extension**
+
+1a. The given index is invalid
+
+> 1a1. Dueue shows an error message <br>
+  Use case resumes at step 1
+
+1b. The task is already "starred"
+
+> 1b1. Dueue shows an error message "This task is already starred." <br>
+Use case ends.
+
+#### Use case: View all starred tasks
+
+**MSS**
+
+1. User requests to view all starred tasks
+2. Dueue displays all tasks being starred <br>
+Use case ends
+
+**Extension**
+
+1a. There is no starred task.
+
+> 1a1. Dueue displays an error message "There is no task being starred." <br>
+  Use case ends.
+#### Use case: View all tasks due in the next n days
+
+**MSS**
+
+1. User requests to view all tasks due in the next n days
+2. Dueue shows a list of tasks in chronological order<br>
+Use case ends.
+
+**Extensions**
+
+1a. The given n is invalid
+
+> 1a1. Dueue shows an error message <br>
+Use case resumes at step 1
+
+2a. The list is empty
+
+> 2a1. Dueue shows message "You have no tasks during that period."<br>
+  Use case ends
+
+#### Use case: See autocomplete messages when typing
+
+**MSS**
+
+1. User requests to list all tasks
+2. Dueue shows a list of tasks
+3. User types "finish" and types part of the name of a very long task name
+4. Dueue displays all the possible tasks <br>
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
+
+> Use case ends
+
+3a. The given name is not part of any task name
+
+> 3a1. Dueue shows message "There is no matching task name" <br>
+  3a2. User edits the name<br>
+  Use case resumes at step 3
+
+#### Use case: Change user interface
+
+**MSS**
+
+1. Advanced user requests to go into settings
```

Comment:
You should ensure there is an equivalent one-step command to change the settings.